<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1190</width>
    <height>649</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="leftMargin">
         <number>20</number>
        </property>
        <item>
         <widget class="QPushButton" name="openFilebtn">
          <property name="text">
           <string>Open File</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>15</height>
             </size>
            </property>
            <property name="text">
             <string>Keyword        :</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="txt_keyword">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>35</height>
             </size>
            </property>
            <property name="placeholderText">
             <string>add keyword(seperate by space)</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>15</height>
             </size>
            </property>
            <property name="text">
             <string>Query Boolean Model:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="txt_query">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>35</height>
             </size>
            </property>
            <property name="placeholderText">
             <string>Gunakan simbol | ( ) &amp; ~</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <property name="sizeConstraint">
           <enum>QLayout::SetMinimumSize</enum>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="checkKeysbtn">
          <property name="text">
           <string>Check Keys</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Boolean Query:</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Hasil Incident Matrix:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lbl_incident_query">
          <property name="text">
           <string>(Buat query)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lbl_incident_result">
          <property name="text">
           <string>-</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Hasil Inverted Index</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lbl_inverted_query">
          <property name="text">
           <string>(Buat query)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lbl_inverted_result">
          <property name="text">
           <string>-</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QTabWidget" name="tabWidget">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tab">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <attribute name="title">
            <string>File</string>
           </attribute>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QTableWidget" name="tbl_preprocess">
              <column>
               <property name="text">
                <string>Filename</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Raw</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Folded</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>No Number</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>No Symbol</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Trimmed</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Tokenized</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Filtered</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Stemmed</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Filepath</string>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <attribute name="title">
            <string>TF/IDF</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <widget class="QTableWidget" name="tf_idf_table">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>40</width>
                <height>321</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>9999999</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="focusPolicy">
               <enum>Qt::NoFocus</enum>
              </property>
              <attribute name="horizontalHeaderCascadingSectionResizes">
               <bool>false</bool>
              </attribute>
              <column>
               <property name="text">
                <string>Keyword</string>
               </property>
               <property name="font">
                <font>
                 <pointsize>12</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
              </column>
              <column>
               <property name="text">
                <string>TF</string>
               </property>
               <property name="font">
                <font>
                 <pointsize>12</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Df</string>
               </property>
               <property name="font">
                <font>
                 <pointsize>12</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
              </column>
              <column>
               <property name="text">
                <string>D/Df</string>
               </property>
               <property name="font">
                <font>
                 <pointsize>12</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
              </column>
              <column>
               <property name="text">
                <string>IDF</string>
               </property>
               <property name="font">
                <font>
                 <pointsize>12</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
              </column>
              <column>
               <property name="text">
                <string>IDF+1</string>
               </property>
               <property name="font">
                <font>
                 <pointsize>12</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
              </column>
              <column>
               <property name="text">
                <string>W</string>
               </property>
               <property name="font">
                <font>
                 <pointsize>12</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
              </column>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QTableWidget" name="tf_idf_result_table_2">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>200</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>11</pointsize>
               </font>
              </property>
              <column>
               <property name="text">
                <string>Document Name</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Similarity</string>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_3">
           <attribute name="title">
            <string>Inverted Index</string>
           </attribute>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QTableWidget" name="inverted_index_table">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>40</width>
                <height>321</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>9999999</height>
               </size>
              </property>
              <property name="focusPolicy">
               <enum>Qt::NoFocus</enum>
              </property>
              <attribute name="horizontalHeaderCascadingSectionResizes">
               <bool>false</bool>
              </attribute>
              <column>
               <property name="text">
                <string>Keyword</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Inverted List</string>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_4">
           <attribute name="title">
            <string>Incident Matrix</string>
           </attribute>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QTableWidget" name="incident_matrix_table">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>40</width>
                <height>321</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>9999999</height>
               </size>
              </property>
              <property name="focusPolicy">
               <enum>Qt::NoFocus</enum>
              </property>
              <attribute name="horizontalHeaderCascadingSectionResizes">
               <bool>false</bool>
              </attribute>
              <column>
               <property name="text">
                <string>Keyword</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Incident Matrix</string>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_5">
           <attribute name="title">
            <string>Jaccard</string>
           </attribute>
           <widget class="QTextEdit" name="txt_jaccard_result">
            <property name="geometry">
             <rect>
              <x>28</x>
              <y>26</y>
              <width>811</width>
              <height>261</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
           <widget class="QTableWidget" name="jaccard_result_table">
            <property name="geometry">
             <rect>
              <x>30</x>
              <y>310</y>
              <width>811</width>
              <height>181</height>
             </rect>
            </property>
            <column>
             <property name="text">
              <string>Document Name</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Similarity</string>
             </property>
            </column>
           </widget>
          </widget>
          <widget class="QWidget" name="tab_6">
           <attribute name="title">
            <string>N-Gram</string>
           </attribute>
           <widget class="QTableWidget" name="n_gram_result_table">
            <property name="geometry">
             <rect>
              <x>30</x>
              <y>310</y>
              <width>811</width>
              <height>181</height>
             </rect>
            </property>
            <column>
             <property name="text">
              <string>Document Name</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Similarity</string>
             </property>
            </column>
           </widget>
           <widget class="QTextEdit" name="txt_ngram_result">
            <property name="geometry">
             <rect>
              <x>28</x>
              <y>46</y>
              <width>811</width>
              <height>241</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
           <widget class="QWidget" name="horizontalLayoutWidget">
            <property name="geometry">
             <rect>
              <x>30</x>
              <y>10</y>
              <width>184</width>
              <height>32</height>
             </rect>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="label_6">
               <property name="maximumSize">
                <size>
                 <width>400</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Input N for Ngram :</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="txt_input_ngram">
               <property name="maximumSize">
                <size>
                 <width>80</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="placeholderText">
                <string>(number)</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
          <widget class="QWidget" name="tab_7">
           <attribute name="title">
            <string>Cosine Sim</string>
           </attribute>
           <widget class="QTextEdit" name="txt_cosine_result">
            <property name="geometry">
             <rect>
              <x>28</x>
              <y>26</y>
              <width>811</width>
              <height>261</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
           <widget class="QTableWidget" name="cosine_result_table">
            <property name="geometry">
             <rect>
              <x>30</x>
              <y>310</y>
              <width>811</width>
              <height>181</height>
             </rect>
            </property>
            <column>
             <property name="text">
              <string>Document Name</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Similarity</string>
             </property>
            </column>
           </widget>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1190</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
